test/consolInterfaceDev.jl

```diff
@@ -7,6 +7,12 @@ using Dates
 using UUIDs
 # using IncrementalInference
 
+## must overload the function
+import DistributedFactorGraphs: getFactorOperationalMemoryType
+# DF, dropping DFG. so that Main context is used and can be overridden by IIF.getFac...Type
+getFactorOperationalMemoryType(::CloudGraphsDFG) = TestCCW
```

Comment:
Sorry, but this is a step back and breaks my testing consolidation effort

Reply:
Think this is now resolved?  We using dispatch on ::AbstractParams rather than ::AbstractDFG as the compromise solution that avoids hard coding a variable somewhere.